tests/phpunit/tests/test-columns.php

```diff
@@ -273,4 +275,25 @@ public function test_post_inline_edit() {
 		$this->assertEquals( 'en', $options->item( 1 )->getAttribute( 'value' ) );
 		$this->assertEquals( 'fr', $options->item( 2 )->getAttribute( 'value' ) );
 	}
+
+	public function test_custom_term_column_for_default_category() {
+		$GLOBALS['taxonomy']     = 'category';
+		$GLOBALS['post_type']    = 'post';
+		$out                     = '';
+		$column                  = 'language_en';
+		$default_cat_id          = get_option( 'default_category' );
+		self::$model->term->set_language( $default_cat_id, 'en' );
+		$column = apply_filters( 'manage_category_custom_column', $out, $column, $default_cat_id );
+
+		$this->assertNotEmpty( $column, 'The generated language column should not be empty.' );
+
+		$doc = new DomDocument();
+		$doc->loadHTML( $column );
+		$xpath = new DOMXpath( $doc );
+
+		$def_cat = $xpath->query( "//div[@id=\"default_cat_{$default_cat_id}\"]" );
+		$this->assertSame( 1, $def_cat->length, 'Only one element with the default category ID should be rendered.' );
+		$def_cat_class_attr = $def_cat->item( 0 )->getAttribute( 'class' );
+		$this->assertSame( 'hidden', $def_cat_class_attr, 'The element of the default category should be hidden.' );
+	}
```

Comment:
Shouldn't we pass this test in `Default_Term_Test` ( see https://github.com/polylang/polylang/blob/master/tests/phpunit/tests/test-default-term.php )? 
The tests for the default terms are here.
I don't know which is better or more consistent.

Reply:
It seems to me that both locations are relevant. What do you think @Chouby ?

Reply:
I would follow Marianne's suggestion.

Reply:
Fine, I've done it in 7f7d285
I leave the `PLL_Admin_Default_Term` instantiation in `Columns_Test` to be sur the column is also filtered for default category. Is it ok for you ?